src/mail/model/MailUtils.ts

```diff
@@ -303,6 +293,60 @@ export function markMails(entityClient: EntityClient, mails: Mail[], unread: boo
 	).then(noOp)
 }
 
+/**
```

Comment:
some little tests for these new functions would be good too

Reply:
Tested! The file name's a little long but it should do the job!